test/protractor.conf.js

```diff
@@ -172,10 +174,10 @@ exports.config = {
   // The timeout in milliseconds for each script run on the browser. This should
   // be longer than the maximum time your application needs to stabilize between
   // tasks.
-  allScriptsTimeout: 11000,
+  allScriptsTimeout: 30 * 1000,//11000,
```

Comment:
Ha, oop, i left the old value in the comment as I was tinkering.  Will eliminate, I think 30 seconds is obvious here.
Math win..?